packages/core/database/src/dialects/cockroachdb/schema-inspector.ts

```diff
@@ -0,0 +1,307 @@
+import type { Database } from '../..';
+import type { Schema, Column, Index, ForeignKey } from '../../schema/types';
+import type { SchemaInspector } from '../dialect';
+
+interface RawTable {
+  table_name: string;
+}
+
+interface RawColumn {
+  data_type: string;
+  column_name: string;
+  character_maximum_length: number;
+  column_default: string;
+  is_nullable: string;
```

Comment:
Could the `column_` fields be renamed since it's clear these are part of the RawColumn interface?

Reply:
I'm wanting to be consistent with pg implementation. Should change for both or neither. Up to core team.